test/functional/conf.js

```diff
@@ -16,5 +18,11 @@ exports.config = {
         require: 'cucumber/step_definitions/*.js',
         tags: ['~@wip', '~@ignore'],
         format: 'progress'
+    },
+
+    onPrepare: function() {
+        var width = 1280;
```

Comment:
For consistency, shouldn't you declare the height/width like you did in the saucelabs version?